package.json

```diff
@@ -29,6 +29,7 @@
   "main": "scripts/index.js",
   "dependencies": {
     "angular-history": "git+https://github.com/decipherinc/angular-history.git#v0.8.0",
+    "superdesk-planning": "git+https://github.com/superdesk/superdesk-planning.git#1f04215",
```

Comment:
Makes sense. Well, now it can be added there, and into that `superdesk.config.js`. So maybe in this case we should not have it here, right?